ubuntu-2004.json

```diff
@@ -32,5 +35,9 @@
       "type": "shell",
       "inline": ["ls /"]
     }
-  ]
+  ],
+  "variables": {
+    "ssh_user": "ubuntu",
+    "ssh_pass": "ubuntu"
+  }
```

Comment:
Could we move the `variables` block to the top of the file? My thinking is that if you were to read from the top to bottom, you'd come across the first variable reference and then have to search the full file to see where it was defined.